Source/Initialization/WarpXInitData.cpp

```diff
@@ -110,9 +111,7 @@ WarpX::InitData ()
 {
     WARPX_PROFILE("WarpX::InitData()");
     Print() << "WarpX (" << WarpX::Version() << ")\n";
```

Comment:
Since you are changing this, can the version information be printed earlier? A number of things happen before `WarpX::InitData` is called and if something goes wrong there, the version info won't be output. A user I'm working with has a case where a problem is happening in `ParseGeometryInput` for example and there is no version info to help debug it.

Reply:
I think we can move our info definitely up. Only the AMReX info will first be printed here, I think.